react/features/base/util/httpUtils.js

```diff
@@ -25,4 +35,17 @@ export function doGetJSON(url) {
 
             return Promise.reject(error);
         });
+
+    if (retry) {
+        return timeoutPromise(fetchPromise, RETRY_TIMEOUT)
+            .catch(response => {
+                if (response.status >= 400 && response.status < 500) {
+                    return Promise.reject(response);
+                }
+
+                return timeoutPromise(fetchPromise, RETRY_TIMEOUT);
+            });
+    }
+
+    return fetchPromise;
```

Comment:
We have this in jitsi-meet-spot as well. You want to shove something into js-utils so both repos can share it?

Reply:
Ok, I will check that.